### DISCUSSION TITLE
Configure Resource limit

### DISCUSSION THREAD

**Discussion by sirmosi** [nextcloud/all-in-one#5615] [Ideas] [answered]
Hi everyone
how does it possible to make resources limit on memory and cpu of nextcloud docker compose?

**Comment by szaimen** [comment, marked as answer]
Hi, this is possible via https://github.com/nextcloud/all-in-one/tree/main/manual-install. See https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml

Additionally, see https://github.com/nextcloud/all-in-one/discussions/7273

**Reply by sirmosi** [reply]
so i have a installed AIO , should i install it again

**Reply by underhax** [reply]
> Hi, this is possible via https://github.com/nextcloud/all-in-one/tree/main/manual-install. See https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml
Hi. Are there any plans to add variables to AIO to customize limits?

**Reply by philiprenich** [reply]
@szaimen Thanks for having the ability to run things manually documented. I found this discussion due to running into the same issue (Recognize app ate all my host resources and made all the services on the server non-responsive for an hour or two).

AIO is done so well, I am loath to go with a manual install and lose out on the automated backups, updates, etc. So I'm also curious if there's a possibility to add memory management variables to the AIO master container that can be passed on to the underlying containers?

**Reply by szaimen** [reply]
Hi, I just created a wiki article for this. See https://github.com/nextcloud/all-in-one/discussions/7273
